### PR TITLE
fix: Use git commands instead of action to perform git merge fast for…

### PR DESCRIPTION
**Pull request by Whiteflight**
# Description

Fix: Use git commands to do merge fast forward

## Checklist

- [x] Changes are limited to a single goal (avoid scope creep)
- [x] I confirm that I have read any Contribution and Development guidelines (CONTRIBUTING and DEVELOPMENT) and are following their suggestions.
- [x] I confirm that I wrote and/or have the right to submit the contents of my Pull Request, by agreeing to the _Developer Certificate of Origin_, (adding a 'sign-off' to my commits).

### CODE REVIEW

**Comment by Whiteflight** [comment]
@dependabot rebase

**Comment by Whiteflight** [comment]
@dependabot recreate

File: .github/workflows/fast_forward.yml [hunk 1]
```diff
@@ -9,22 +9,69 @@ on:
...
+          BASE_BRANCH: ${{ fromJson(steps.get-pr-details.outputs.data).base.ref }}
+          HEAD_BRANCH: ${{ fromJson(steps.get-pr-details.outputs.data).head.ref }}
+        run: |
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "GitHub Actions"        
+          git checkout $HEAD_BRANCH
+          git checkout $BASE_BRANCH
+          git merge $HEAD_BRANCH --ff-only
+          git push
+          echo "::set-output name=message::'Fast forward succesfull.'"
```
Comment (Racemovie) [inline .github/workflows/fast_forward.yml:54]: successful*

File: .github/workflows/fast_forward.yml [hunk 2]
```diff
@@ -9,22 +9,69 @@ on:
...
+          pull_number: ${{ github.event.issue.number }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      # Merge (rebase) the PR if it is allowed.
+      - name: Fast forward the PR
+        id: merge-status
+        shell: bash
+        env:
+          BASE_BRANCH: ${{ fromJson(steps.get-pr-details.outputs.data).base.ref }}
```
Comment (Racemovie) [inline .github/workflows/fast_forward.yml:45]: lägg dem i samma ordning som de används sen

File: .github/workflows/fast_forward.yml [hunk 3]
```diff
@@ -9,22 +9,69 @@ on:
...
+
+      # Post a failure message when any of the previous steps fail.
+      - name: Add failure comment to PR
+        if: ${{ failure() }}
+        uses: octokit/request-action@v2.3.1
+        with:
+          route: POST /repos/{repository}/issues/{issue_number}/comments
+          repository: ${{ github.repository }}
+          issue_number: ${{ github.event.issue.number }}
+          body: PR cannot be fast forward. Check the Actions execution tab for details.
```
Comment (Racemovie) [inline .github/workflows/fast_forward.yml:75]: forwarded*? eller "Unable to fast forward"?

**Review by Racemovie — CHANGES_REQUESTED** [review]

File: .github/workflows/fast_forward.yml (hunk 1 above)
Comment (Whiteflight) [reply .github/workflows/fast_forward.yml]: Fixat

File: .github/workflows/fast_forward.yml (hunk 2 above)
Comment (Whiteflight) [reply .github/workflows/fast_forward.yml]: Fixed

File: .github/workflows/fast_forward.yml (hunk 3 above)
Comment (Whiteflight) [reply .github/workflows/fast_forward.yml]: Fixed

**Review by Racemovie — APPROVED** [review]
LGTM

**Pull request merged**